tests/unit/bokeh/core/property/test_container.py

```diff
@@ -286,6 +287,13 @@ def test_str(self) -> None:
         prop = bcpc.Tuple(Int, Int)
         assert str(prop) == "Tuple(Int, Int)"
 
+class Test_RestrictedDict:
+    def test_invalid(self) -> None:
+        prop = bcpc.RestrictedDict(String, bcpc.List(Int), disallow=("disallowed_key_1", "disallowed_key_2"))
+
+        assert not prop.is_valid({"disallowed_key_1": [1,2,3]})
+        assert not prop.is_valid({"disallowed_key_2": [1,2,3]})
+
```

Comment:
If we want to be completely thorough, it would be good to also verify somewhere that non-disallowed keys *do* work as well.